.travis.yml

```diff
@@ -20,8 +21,6 @@ cache:
   directories:
     - $HOME/.composer/cache/files
 
-before_install: composer self-update
```

Comment:
Why?

Reply:
Duplicated, it is already in the line below